theia-full-docker/Dockerfile

```diff
@@ -80,36 +80,35 @@ RUN apt-get -y install git sudo
 #LSPs
 
 ##GO
-ENV GO_VERSION 1.11.4
+ENV GO_VERSION 1.12
 ENV GOPATH=/usr/local/go-packages
 ENV GO_ROOT=/usr/local/go
 ENV PATH $PATH:/usr/local/go/bin
 ENV PATH $PATH:${GOPATH}/bin
 
-RUN curl -sS https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf - && \
-    go get -u -v github.com/nsf/gocode && \
-    go get -u -v github.com/uudashr/gopkgs/cmd/gopkgs && \
-    go get -u -v github.com/ramya-rao-a/go-outline && \
-    go get -u -v github.com/acroca/go-symbols && \
-    go get -u -v golang.org/x/tools/cmd/guru && \
-    go get -u -v golang.org/x/tools/cmd/gorename && \
-    go get -u -v github.com/fatih/gomodifytags && \
-    go get -u -v github.com/haya14busa/goplay/cmd/goplay && \
-    go get -u -v github.com/josharian/impl && \
-    go get -u -v github.com/tylerb/gotype-live && \
-    go get -u -v github.com/rogpeppe/godef && \
-    # godoc causing issues ATM, preventing image to build - commenting-out
-    # go get -u -v golang.org/x/tools/cmd/godoc && \
-    go get -u -v github.com/zmb3/gogetdoc && \
-    go get -u -v golang.org/x/tools/cmd/goimports && \
-    go get -u -v sourcegraph.com/sqs/goreturns && \
-    go get -u -v github.com/golang/lint/golint && \
-    go get -u -v github.com/cweill/gotests/... && \
-    go get -u -v github.com/alecthomas/gometalinter && \
-    go get -u -v honnef.co/go/tools/... && \
-    go get -u -v github.com/sourcegraph/go-langserver && \
-    go get -u -v github.com/derekparker/delve/cmd/dlv && \
-    go get -u -v github.com/davidrjenni/reftools/cmd/fillstruct
+RUN curl -sS https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf -
```

Comment:
these are the same as in the go-specific image?

Reply:
Yes, I tried the entries from `theia-go` and also updated the go version.
The error looks to be narrowed down to `golint` and `go-langserver` for now.

Reply:
Yes. I was also preparing a PR to solve this. Instead of commenting golint you can update the path to "golang.org/x/lint/golint".

In addition, it doesn't hurt update the go version to 1.14 on both theia-full and theia-go. I have done it and tested on my branch with success.

Reply:
> Yes. I was also preparing a PR to solve this. Instead of commenting golint you can update the path to "golang.org/x/lint/golint".
> 
> In addition, it doesn't hurt update the go version to 1.14 on both theia-full and theia-go. I have done it and tested on my branch with success.

Thanks, I was commenting out the failures to identify which exact commands were failing and then work towards resolving them.